Sources/Utilities.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+struct Utilities {
```

Comment:
Yeah, that's a valid point. In that case, how about a `StringEncoder` class or something like that?